docs/user/assets/metrics-resources.drawio.svg

```diff
@@ -1,4 +1,482 @@
fe6PJCXDV1rOyuDB1tz+iJQE86ORrREtcHWluphtB6I+8rBUfhSD05E3J0LT9nq3T0Ppj0NivMVLpIEQmo2UuN71Tq0IzghXZtzk4KbLYSZuphxy+FwsilpKmYHfX3GzfJ635V7iyC09BUa5lJPWBYYPfQBW+BykhGJLYcdSWBWwjpS5LT0oub0WvKy1/OzraYcs+z7hYyY/V6tZe8gZS1HnDwMdx43tutyKNgaP6E8DA7euJcs4VIVScOXcL2LK/B9jfVR0pe2/Q6C2WW3J0nsatGTSWBBXJUOvawa8Trn84OUT81NKcvw4DgpRfyI68VbqY/rNhobJYiEEA9RI85fz5lkfffNmjBJOiLgHhSYFjnQSD9ee8wo7sAPp8wWpJhFCqj8AxfavyDgB1YO2ZqZmxSwBMPQY1HlZoVbvyrCxR84v6m7brpxiwwoYOR8F2GYJ4TECJxFjv8cKRzOt59YlkIpksZxjCZwjDje3odOERyrUOwTS1A4hmAUcIYx8lghcEM/ODUKI54oEvwQSmM0S1PnRoIQpL3tPzcK2uxjU2HupebdOD63vP2OIaFzfIakTh0149I9Fv3AKHOzBaMSF+3QvL4S3G2cHjZwmM7l/kWt81Vb28oA5bw4xEfK1K5MYbGXJsXPPGxAKfDsEBTf1seHTvfBt5ZMAaL3tkM/N2Zi+m52bOoLQSxLy8RxYVdQUH8fhIWrbk0b3t1nJmxPUGzi0+28yNLI5dM4zdqncar9nBqktu0RUPxRcCCQCwBgJPKG4yiGvGUyhTyAq8yboXId31VPl2lWBKmfJmYsvpS+Hiq3DgsNfEeeyNOVDq9O34X69cXhfJGARh4fwkn0XNA+B6DPngteHm6vnp+OrXQPvsMRDm0TNIS7ZO597rWEJm4WgmFysxO9uyzM0zKz3UvwF2bmu6dqzKkeHK93GZ25sVmElXtB/RbTTo9O0hBKzTNAUPoKIBT+xFLIy4e4JHls+InKCxTeEiaQS8IoRV5SOnb3DaVOlpmHV9W2sEL+Xg+ufghDkfdb9osH4NRl24YXjD+P94dgz+BvlNlRc4BeIHHoufbBBlC/JRoD03LjSyEw49BPwHcbQAjiijtjs3O6sQkdp5WczAXazLRaehB8p5EDxEnuBwkNTAzJc6Yd+S2qzzBO0sS9CdazAL9ROBhip0ni2sXp50AZ8q4i+ok8oTR6GpgvYpa8ZN/PK4Smnpe7xY9rPfU5HpLspwzSIPUn4daNQzCQ32+R7DIv0s3XzJBj5sEHtNq1TfLCOH5VT6Tgn2+yVTjzB23VmcanbNXliP/Scp3v/NJykRd26/M26z4/f9sgnQXi+w0SeoUChHmMQcII7Im8IE0iHzNJv6E/qDP8X/TH3izswM3/Pr3/Wsaxmxg5C8fX7QDyhODsJRfQhxgFHL00Cij1DVaBwj5lFYaAcmj/1zB82TBcWXwQ1v5Bw/DWm3vm1k1m2cdRgIzKfOt/QRPAbyDn//6vHRPkmqPkHY4uchhvIiNz4+Yty14BhbqAzo/2rfPnsq8g4B4r34BBoOCfG6rJSguIw8dwH6OIJ5a+BAB9AwD0DQA8G5MvIYD4lNSrrp25fyIx8XBp/44EBMb8k04d+V/O/T7n2H9S6/5O6uj/kTt+1kn/XnccEH5CMPb5c6nhCewKRN+VLcKJX2SL8Ouo9PKBL2eLKPY3kH4D2+grZL/g/FfYvkD2C9DvYPuDvuGHUH32pS9QTf4ZVGPXuUkahXHJy+cqePgoqPFP0n1cvHl2fm54qdA2fMhaYXCD8LWpylzPBXy0YeD67GgeSd7xNf/xePYt/M4y9pD4lSEvVdXPE5C/iMlzGvM5gMUuKTwkgKVvOkQtW0F4kXwxnvHMTRgfjlV7bly5kL+3452zZ3S6cfxReCdJs01rNM/3KjMLTfA/QIlZlAAbv6hnm9t7VfYnJwXehNMT7Z3YLQAef8LoKkz8t0+m2TYwkxPJY3oEASAqfobA10hO1JBzd9o7RQYe8ACNM7UTLKHnlO4vSe3TzLn88WdaoL1WFAJykOZR2/48+V5vf9Nx7TQD+EuTn0UQ2lHi5qcfCpOwCM+9ua77auTfrfcKBBf1vDg1i+tuOmG+jc3Dufox+YH8T7jZAmtmJsVNn3iSpUDJ5O0kyTFx8lrjHNF5R+PAxv7aYb60V6fWXrqMV5Gv5zmEhd/SXY/wr3HsOht5y7++Mf+OP8C/Zt7mJ98M6+PCkdfjeldJf3oEGfzKMyOfKPKVZ4pQb8aTvpkkoB8woGej8NEVDKa7SRP1j0SbrxYwtHL10/Tdk8PxB1YxXGeTeJ6R2AeJ0HViEL0pQt8Vot5cs9IyzAmrL3Hs064bfst1ew9/fJp4oT8EbP0A/t54fM/FbUcvS8MvNfjd4bv/xO0ftZ49xHv266Oi8tzh8GODcFnzK4L2xvT8Rtj1CZVKfECgiO8SqJsq9H1BeCSI7nrAnxXb1yB5ULLxY31rvbYTYjqtb+v9Q3IDfHV3DzMzX5OcO/7fewRuzoy8G7A+yPA9wqA9p70Ol2L1WvzgAk36eySQRt8Mzu8t2PvxG7moq8V6fzIXdXbeLnJRJ/P+/RlWlnwimJccEXPp1F4nQj+cYMXRT9F9YC4K+7we/1AuagMEtcjfCjrc/IhUaVxu3I/K/F+YpKLvqYu/KEl1naNiLgk8JkeFP0YJfSYh/khlQt+arqH/lDKh3xN6lPxdZYJcTeO+ofSo+Rn8atYRxX81P3P1APvY6Rn65uT/5/xF7I6/+Da3ftN5ee37tKrOTap/hUI7r5z5ixXaT5y6Umnfknb/V2xvefbZ//2JIQy5WmP4JxNDNP3nDdjfMKN7y/DRf2zZ8PXUK/Lbpg77BaW/xtShJw17fw/OdVcuH/i6cby5S+K/KdAvNfivSYE+KIfz78l+vpUXgnxrNm5NIZFXGwk+YDbAZZbCkX+RPdD9YJg60IcT/wM=&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
why is "Processes Metrics" orange, but no box in the diagram is orange?

Reply:
In this context, does it really matter whether something is a DaemonSet or a Deployment etc? I'd say it just makes it harder to read. 
How relevant is this level of detail to our readers, also in Help Portal? 
Many other diagrams work well with human-readable text (see my other comment on 01-manager.md).

Reply:
fixed coloring. As discussed I like to keep the resource types in the diagram